### ISSUE TITLE
Failed to create SVG react component 

### ISSUE THREAD

**Issue by nen-io** [Danzo7/snowpack-svgr-plugin#1]
![image](https://user-images.githubusercontent.com/32480489/148251216-753f0d20-8864-440c-9a77-83535444e4a1.png)

seems like babel expects a `filename`. 
![image](https://user-images.githubusercontent.com/32480489/148251320-d208ec9a-35c7-4759-8787-81fbeb7c0823.png)

I've tested locally by setting the `filename:filepath` as an option past into babel at it seems to work fine.

**Comment by Danzo7** [comment]
Hello @ajith1234 ,I just try to reproduce on a starter project and I was not able to get the error its work just as intended, can you provide a repo with the error above?.

Its stated in the [docs ](https://babeljs.io/docs/en/options#filename )that `filename` is only required if a plugin requires it.
> The filename is exposed to plugins. Some plugins may require the presence of the filename.

@babel/preset-react stated that it does not, as In[ `@babel/preset-react`](https://babeljs.io/docs/en/babel-preset-react/#via-node-api) the example of usage in node API is:
```
require("@babel/core").transformSync("code", {
  presets: ["@babel/preset-react"],
});
```
which is the same as I use (no `filename` option).
As its seems to work fine on my end we need to reproduce the error first before merge the fix,Thanks for the effort!

**Comment by nen-io** [comment]
hey thanks for the quick response,

I cant share the repo as its a large TS project which im working on and having to migrate from webpack. This was the first time working with snowpack, babel plugins etc, so apologies for the noob questions or attempted solution.

Turns out I had a issue with having babel plugin for snowpack and yours leading to some conflicting issues with filename. I've just removed babel plugin as it dont think its actually even needed for my this project.
